Makefile

```diff
@@ -1,12 +1,21 @@
 .PHONY: all test clean
 VENV_PYTHON:=venv/bin/python
+TENV_PYTHON:=test_venv/bin/python
 
 all: test
 
+build:
+	python setup.py bdist
+	@touch $@
+
 venv: setup.py
 	rm -rf $@
 	python -m venv $@
 
+test_venv: setup.py
```

Comment:
We could make this DRY if we used [Pattern Rules](https://www.gnu.org/software/make/manual/html_node/Pattern-Rules.html). But that would require changing the names `venv` to `.venv` and test_venv` to `test.venv`.